test/utils.sh

```diff
@@ -359,6 +381,87 @@ get_unreleased_bundle() {
 
 }
 
+# This function will be used by tekton tasks in build-definitions
+# It returns a list of unreleased related images as indicated in a FBC fragment.
+# It compares the provided FBC fragment against the corresponding production index image
+get_unreleased_fbc_related_images() {
+  # FBC fragment containing the unreleased bundle
+  local FBC_FRAGMENT=""
+  local INDEX_IMAGE="registry.redhat.io/redhat/redhat-operator-index"
+
+ get_unreleased_fbc_related_images_usage()
+  {
+    echo "
+  get_unreleased_fbc_related_images  -i FBC_FRAGMENT [-b INDEX_IMAGE]
+  " >&2
+    exit 2
+  }
+
+  local opt
+  while getopts "i:b:" opt; do
+      case "${opt}" in
+          i)
+              FBC_FRAGMENT="${OPTARG}" ;;
+          b)
+              INDEX_IMAGE="${OPTARG}" ;;
+          *)
+              get_unreleased_fbc_related_images_usage ;;
+      esac
+  done
+
+  if [ -z "$FBC_FRAGMENT" ]; then
+    echo "Missing parameter FBC_FRAGMENT" >&2
+    exit 2
+  fi
+
+  # If the index image is provided and has a tag, remove it.
+  # The target ocp version is determined from the fragment
+  if [[ "$INDEX_IMAGE" == *:* ]]; then
+    INDEX_IMAGE="${INDEX_IMAGE%%:*}"
+  fi
+
+  #Get target ocp version from the fragment
+  local ocp_version
+  if ! ocp_version=$(get_ocp_version_from_fbc_fragment "$FBC_FRAGMENT"); then
+    echo "Could not get ocp version for the fragment" >&2
+    exit 1
+  fi
+
+  # Run opm render on the FBC fragment to extract package names
+  local render_out_fbc unique_bundles_fbc package_names
+  if ! render_out_fbc=$(opm render "$FBC_FRAGMENT"); then
+    echo "Could not render image $FBC_FRAGMENT" >&2
```

Comment:
Can we have different wording either for this output or the one on line 442, something like:
`Could not render image from FBC-FRAGMENT <FBC-FRAGMENT>. So its obvious from first look when we receive this error msg?

Reply:
I'm fine with the ned goal.

Reply:
This has been addressed in https://github.com/konflux-ci/konflux-test/pull/356